src/component/mxgraph/StyleConfigurator.ts

```diff
@@ -100,30 +98,14 @@ export default class StyleConfigurator {
     this.graph.getStylesheet().putCellStyle(ShapeBpmnElementKind.LANE, style);
   }
 
-  private configureStartEventStyle(): void {
-    const style = this.cloneDefaultVertexStyle();
-    style[this.mxConstants.STYLE_SHAPE] = ShapeBpmnElementKind.EVENT_START;
-    style[this.mxConstants.STYLE_PERIMETER] = this.mxPerimeter.EllipsePerimeter;
-    style[this.mxConstants.STYLE_VERTICAL_ALIGN] = 'top';
-    this.putCellStyle(ShapeBpmnElementKind.EVENT_START, style);
-  }
-
-  private configureEndEventStyle(): void {
-    const style = this.cloneDefaultVertexStyle();
-    style[this.mxConstants.STYLE_SHAPE] = ShapeBpmnElementKind.EVENT_END;
-    style[this.mxConstants.STYLE_PERIMETER] = this.mxPerimeter.EllipsePerimeter;
-    style[this.mxConstants.STYLE_VERTICAL_ALIGN] = 'top';
-    style[this.mxConstants.STYLE_STROKECOLOR] = '#000';
-    this.putCellStyle(ShapeBpmnElementKind.EVENT_END, style);
-  }
-
-  private configureEndEventTerminateStyle(): void {
-    const style = this.cloneDefaultVertexStyle();
-    style[this.mxConstants.STYLE_SHAPE] = ShapeBpmnElementKind.EVENT_END;
-    style[this.mxConstants.STYLE_PERIMETER] = this.mxPerimeter.EllipsePerimeter;
-    style[this.mxConstants.STYLE_VERTICAL_ALIGN] = 'top';
-    style[this.mxConstants.STYLE_STROKECOLOR] = '#F00';
-    this.putCellStyle(ShapeBpmnElementKind.EVENT_END + '_' + ShapeBpmnEventKind.TERMINATE, style);
+  private configureEventsStyle(): void {
+    ShapeUtil.topLevelBpmnEventKinds().forEach(kind => {
+      const style = this.cloneDefaultVertexStyle();
+      style[this.mxConstants.STYLE_SHAPE] = kind;
+      style[this.mxConstants.STYLE_PERIMETER] = this.mxPerimeter.EllipsePerimeter;
```

Comment:
If the **EventShape** extends **mxEllipse**, maybe we don't need that anymore.

Reply:
yes probably, I also saw that but didn't take time to check if this is required

Reply:
@csouchet I had a look at this, removing it doesn't change anything in the display.
However, all mxGraph examples set this property and at 1st glance, this seems useful to define borders of a shape which may be interesting in the future, see https://jgraph.github.io/mxgraph/docs/js-api/files/view/mxPerimeter-js.html#mxPerimeter
So I suggest to keep it for now